### PR TITLE
fix(plugins): fix multiple bugs to current version

### PR DESCRIPTION
**Pull request by warren830**
# Summary
1. 704b291bb3ab3ae46fe53ba09207cbda65886078 use generated id for pull_request_id in pull_request_commits
2. d37c84ac940f9247f2404d364fee3567ed535769  added issue_labels to domainLayer
3. 692d61f98abe20f09660e909596b922f26883c38 changed "triggers" in menu sidebar to "Triggers"
4. a0e6965f646da2a134d8bee5f0e04d7a165c76a8  added relationship between issue and repo in table.board_issues, and store the info of repo in table.boards
5. 8975340f1f39ec961c24c843e6a8b2d02932287b changed value of type, status in table.issues to uppercase
6.19af70a06b5247f12a2b9966d9eec5f2d068784e  let issue.assigneeId be users.id, added assigneeName to table.issue
7. 593399de8f66225a4215b6f671f0c698d8d2fbb2 fix issue #1092 
### Screenshots
for 1: use generated id for pull_request_id in pull_request_commits
image

for 2: added issue_labels to domainLayer
image

for 3: changed "triggers" in menu sidebar to "Triggers"
image

for 4: added relationship between issue and repo in table.board_issues, and store the info of repo in table.boards
image
image

for 5: changed value of type, status in table.issues to uppercase
image

for 6: let issue.assigneeId be users.id, added assigneeName to table.issue
image

### CODE REVIEW

File: plugins/github/tasks/github_issue_converter.go [hunk 1]
```diff
@@ -2,51 +2,64 @@ package tasks
...
 	}
 
 	return nil
 }
 
 func convertStateToStatus(state string) string {
 	if state == "closed" {
-		return "Resolved"
+		return ticket.DONE
+	} else if state == "todo" {
```
Comment (klesh) [inline plugins/github/tasks/github_issue_converter.go:44]: github issue status either `open` or `closed`, no `todo` at all, we want they are mapped to `TODO` and `DONE`

Comment (warren830) [reply plugins/github/tasks/github_issue_converter.go:44]: ok，fixed:
func convertStateToStatus(state string) string {
	if state == "closed" {
		return ticket.DONE
	} else {
		return ticket.TODO
	} 
}

File: plugins/github/tasks/github_issues_collector.go [hunk 2]
```diff
@@ -85,7 +85,8 @@ func convertGithubIssue(issue *IssuesResponse, repositoryId int) (*models.Github
 		State:           issue.State,
 		Title:           issue.Title,
 		Body:            issue.Body,
-		Assignee:        issue.Assignee.Login,
+		AssigneeId:      issue.Assignee.Id,
```
[the remark this answers is not in the document]
Comment (warren830) [reply plugins/github/tasks/github_issues_collector.go:88]: how about like this:
if &issue.Assignee != nil {
    githubIssue.AssigneeId = issue.Assignee.Id
    githubIssue.AssigneeName = issue.Assignee.Login
}

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**